test/lib/generateToken.test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const assert = require('chai').assert;
+const mockery = require('mockery');
+const sinon = require('sinon');
+
+sinon.assert.expose(assert, { prefix: '' });
+require('sinon-as-promised');
+
+describe('generateToken', () => {
+    let nodeifyMock;
+    let generateToken;
+    const randomBytes = 'some random bytes';
+    // Can't mock crypto because code coverage tool uses it too
+    // This is what a sha256 hash turns the random bytes into
+    const hash = 'mF0EvjvxWMrVz5ZGJcnbe0ZPooUlv/DAB9VrV6bmZmg=';
```

Comment:
Let's consider it a run-time dependency (like `async`) since it can't be mocked out. You should still inspect that it was called with the expected parameters.